Program.cs

```diff
@@ -1,10 +1,18 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Convert decimal to hexadecimal and binary
+
+// To Hexadecimal
+
+Console.Write ("Enter a decimal value: ");
+int input = int.Parse (Console.ReadLine ());
+string hexvalue = input.ToString ("X");
+Console.WriteLine (hexvalue);
+
+// To binary 
+
+string binarynumber = "";
+while (input > 0) {
+   int remainder = input % 2;
+   binarynumber = Convert.ToString (remainder) + binarynumber;
+   input = input / 2;
+}
+foreach (var item in binarynumber) Console.Write (item);

```

Comment:
No need a for each loop this is a string not a collection you can pass the string to Console.Write method.
Mean while try to test with more possible options before committing the code.